pyre/extra_tests/parity_tests/raise_from_invalid_cause.py

```
@@ -0,0 +1,114 @@
+        raise ValueError("cause constructor")
+
+
+def invalid_cause_still_builds_the_raised_class():
+    """`raise Raised from 42` runs `Raised()`, then rejects the cause."""
+    before = Raised.constructed
+    try:
+        raise Raised from 42
+    except TypeError as error:
+        message = str(error)
+    else:
+        raise AssertionError("an invalid cause must raise TypeError")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)